src/parser/demonhunter/havoc/modules/talents/Felblade.js

```diff
@@ -69,13 +75,15 @@ class Felblade extends Analyzer {
           <>
             {effectiveFuryGain} Effective Fury gained<br />
             {this.furyGain} Total Fury gained<br />
-            {this.furyWaste} Fury wasted
+            {this.furyWaste} Fury wasted<br />
+            {formatThousands(this.damage)} Total damage
           </>
         )}
       >
         <BoringSpellValueText spell={SPELLS.FELBLADE_TALENT}>
           <>
-            {this.furyPerMin} <small>Fury per min </small>
+            {this.furyPerMin} <small>Fury per min </small><br />
+            {this.owner.formatItemDamageDone(this.damage)}
           </>
```

Comment:
You can also use <ItemDamageDone amount={this.damage}/> for a nice added sword to the metric to match rest of the codebase

Reply:
Oh, looks better!